community/detectors/bitbucket_cve_2022_36804/src/main/java/com/google/tsunami/plugins/detectors/bitbucket/Cve202236804VulnDetector.java

```diff
@@ -0,0 +1,212 @@
+package com.google.tsunami.plugins.detectors.bitbucket;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static com.google.tsunami.common.data.NetworkEndpointUtils.toUriAuthority;
+import static com.google.tsunami.common.net.http.HttpRequest.get;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.flogger.GoogleLogger;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.google.protobuf.util.Timestamps;
+import com.google.tsunami.common.data.NetworkServiceUtils;
+import com.google.tsunami.common.net.http.HttpClient;
+import com.google.tsunami.common.net.http.HttpResponse;
+import com.google.tsunami.plugin.PluginType;
+import com.google.tsunami.plugin.VulnDetector;
+import com.google.tsunami.plugin.payload.Payload;
+import com.google.tsunami.plugin.payload.PayloadGenerator;
+import com.google.tsunami.plugin.annotations.PluginInfo;
+import com.google.tsunami.proto.*;
+import com.google.tsunami.common.time.UtcClock;
+
+import java.io.IOException;
+import javax.inject.Inject;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLEncoder;
+import java.time.Clock;
+import java.time.Instant;
+import java.util.Iterator;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * A {@link VulnDetector} that detects the CVE-2022-0540 vulnerability. Reading
+ */
+@PluginInfo(
+        type = PluginType.VULN_DETECTION,
+        name = "Cve202236804VulnDetector",
+        version = "0.1",
+        description =
+                "A vulnerability in Bitbucket allows a remote, An attacker with access "
+                        + "to a public Bitbucket repository or with read permissions to a"
+                        + "private one can execute arbitrary code by sending a malicious "
+                        + "HTTP request. This All versions released after 6.10.17 "
+                        + "including 7.0.0 and newer are affected, this means that all "
+                        + "instances that are running any versions between 7.0.0 and "
+                        + "8.3.0 inclusive can be exploited by this vulnerability.",
```

Comment:
Please update the description according to the changes you are doing in the README from my other comments.